docs/ExperimentConfig.md

```diff
@@ -160,6 +164,14 @@ machineList:
     __useAnnotation__ means whether you use annotation to analysis your code and generate search space. 
 	   
 	    Note: if you set useAnnotation=True, you should not set searchSpacePath.
+
+* __nniManagerIp__
+  * Description
+   
+    __nniManagerIp__ set the IP address of your machine which the nni manager process runs on. This field is an optional choice, if you don't set nniManagerIp, nni will use the IP of etho device.
```

Comment:
set the IP address of the machine on which nni manager process runs. This field is optional, and if it's not set, eth0 device IP will be used instead.

Reply:
updated to new format